js/tasks.js

```diff
@@ -0,0 +1,8 @@
+import { appearById, hideById } from "./modules/login.js" 
+
+window.appearById = function() {
+    appearById('modalContainer')
+}
```

Comment:
if you do this like that, you won't be able to show anything else by id, other than `modalContainer`. The point of parameterising functions is to able to reuse them with different parameters. In this case, this would be more usable like this:

```suggestion
window.appearById = function(id) {
    appearById(id)
}
```

and sending the id in the html. In that way you can use this function in different places in your html with different elements and they all will behave the same way